<launch>

  <!-- CON-SAI2 vs CON-SAI2 用の引数 -->
  <!-- 設定されるとconsai2_descriptionのパラメータを上書きする -->
  <arg name="side" default="" />
  <arg name="color" default="" />

  <!-- その他パラメータの読み込み -->
  <include file="$(find consai2_description)/launch/description.launch" >
    <arg name="side" value="$(arg side)" />
    <arg name="color" value="$(arg color)" />
  </include>

  <include file="$(find consai2_receiver)/launch/receiver.launch" >
    <arg name="load_description" value="false" />
  </include>

  <node name="vision_wrapper" pkg="consai2_world_observer" type="consai2_world_observer" required="true" output="screen" />

  <node name="referee_wrapper" pkg="consai2_game" type="referee_wrapper.py" required="true" output="screen" />

  <node name="consai2_visualizer" pkg="consai2_gui" type="consai2_gui" required="true" output="screen" />

</launch>
